types/client.d.ts

```diff
@@ -0,0 +1,19 @@
+import { Person } from "./person";
+import { Project } from "./project";
+
+export interface Client {

```

Comment:
Create the `ClientPatch` interface that has all of the fields as optional.
Client the `ClientCreate` with the definition of mandatory and non-mandatory fields.